lesson4/test_api/test_updating_resource_with_patch.py

```diff
@@ -0,0 +1,50 @@
+# Проверяем валидные id ресурсов
+def test_update_resource_with_patch_positive_id(session, base_url, fixture_id_resource_positive):
+    payload = {"title": "some title"}
+    res = session.patch(url=f'{base_url}/{fixture_id_resource_positive}', json=payload)
+    assert res.status_code == 200, "Status code is not 200"
+    assert res.json()["id"] == fixture_id_resource_positive, "Id is different"
+    assert res.json()["title"] == payload["title"], "Wrong title"
+
+
+# Проверяем замену значений всех ключей
+def test_update_resource_with_patch_positive_all_values(session, base_url):
+    payload = {"title": "NEW Title", "completed": False, "userId": 10}
```

Comment:
As I can see, the original value of "completed" key for todo with id = 1 is false
```
curl https://jsonplaceholder.typicode.com/todos/1
{
  "userId": 1,
  "id": 1,
  "title": "delectus aut autem",
  "completed": false
}
```
Please, replace the value of "completed" key to True to prevent updating todo with the same value.